src/Resources/views/Form/Type/sonata_type_model_autocomplete.html.twig

```diff
@@ -157,7 +161,11 @@ file that was distributed with this source code.
                     item.label = item.text;
                 }
                 return {% block sonata_type_model_autocomplete_selection_format -%}
-                    jQuery('<div>').text(item.label).prop('innerHTML')
+                    {% if safe_label|default(false) %}
+                        item.label
```

Comment:
what about the div here. is it not needed?

Reply:
It seems the div never was here

Reply:
you are right